src/components/ResourceForm/AmbulanceFormPage.tsx

```diff
@@ -19,7 +19,7 @@ import DoneButton from './DoneButton';
 import { Colours } from '../../styles/Constants';
 
 const useStyles = makeStyles({
-  resourceWrapper: {
+  resourcewrapper: {
```

Comment:
the issue was with pascalCase not the word wrapper 😅
```suggestion
  resourceWrapper: {
```